garnix.yaml

```diff
@@ -0,0 +1,3 @@
+builds:
+  exclude:
+    - "*"
```

Comment:
Might as well remove the Garnix app from the repo instead

Reply:
This is just to save some cycles, I'll drop this and the garnix app can be disabled when we're ready to merge.